core/templates/dev/head/pages/skill_editor/skill_editor.html

```diff
@@ -233,6 +233,9 @@
   <script src="/extensions/objects/templates/SetOfUnicodeStringEditorDirective.js"></script>
   <script src="/extensions/objects/templates/UnicodeStringEditorDirective.js"></script>
 
+  <script src="/templates/dev/head/services/IdGenerationService.js"></script>
+  <script src="/templates/dev/head/services/HtmlEscaperService.js"></script>
```

Comment:
Move this above the IdGenerationService import (alphabetical order).

Reply:
done